### DISCUSSION TITLE
interactive Flash pages

### DISCUSSION THREAD

**Discussion by ahmedabbous** [TDRA-ae/aegov-dls#34] [General]
Hello,

Is it ok to have pages like the following:
https://www.fahr.gov.ae/portal/Userfiles/Assets/Magazines/Online/35334b8e/

its flash based page

Thank you

**Comment by moiz-conceptlz** [comment]
Hello @ahmedabbous 

Our understanding is that a flipbook tool was used to generate an interactive version of the magazine from a PDF file. We also:

- tested this link on various browsers, it works on all major browsers
- tested this link on various mobile devices, it works on iOS and Android. Perhaps some older version don't support the effect of flipping, however it is still usable.

Here is our suggestions:

1. Embed the tool on a page so that you are able to replicate the similar look as something similar to https://issuu.com/primecreativemedia-2016/docs/energy_0624_lr. This way, the header, footer and other elements you may want to add on the page are still visible and the user may navigate away from this page to any other section of the website with ease.
2. The call to action - where the user may click on button that leads to this page, must also have a call to action to download the PDF version. The reason for this is that although the flipping book is working well on browsers and desktops, it may not pass accessibility testing. Hence, **an alternate version** must be available - such as a PDF that can be used by screen readers to read out the content of the PDF.
3. It is always a good practice to design the page with some additional elements such as a title of the magazine, the date it was added, if there are related magazine then to show a list of that as well, allow the ability to share this link on social media or with other users etc. Hence you may be creative on the layout of the page - aim to use icons and components from the design system on this page. All of this is optional, but will contribute to a better experience.
